From 59baff1aeb6b4f6e3c6e838f6364cde0acb80c71 Mon Sep 17 00:00:00 2001
From: Bruno Haible <bruno@clisp.org>
Date: Sat, 14 Sep 2019 13:30:24 +0200
Subject: [PATCH 8/9] build: Avoid failing a configure test due to
 CFLAGS="-Werror".

Reported by Florian Weimer <fweimer@redhat.com>.

* gettext-runtime/m4/intdiv0.m4 (gt_INTDIV0): In the test program, include
<unistd.h>, for the declaration of _exit().
---
 gettext-runtime/m4/intdiv0.m4 | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/gettext-runtime/m4/intdiv0.m4 b/gettext-runtime/m4/intdiv0.m4
index e63fcf100..3dbc944f7 100644
--- a/gettext-runtime/m4/intdiv0.m4
+++ b/gettext-runtime/m4/intdiv0.m4
@@ -1,4 +1,4 @@
-# intdiv0.m4 serial 6 (gettext-0.18.2)
+# intdiv0.m4 serial 7 (gettext-0.21)
 dnl Copyright (C) 2002, 2007-2008, 2010-2019 Free Software Foundation, Inc.
 dnl This file is free software; the Free Software Foundation
 dnl gives unlimited permission to copy and/or distribute it,
@@ -31,8 +31,11 @@ changequote([,])dnl
       if test -z "$gt_cv_int_divbyzero_sigfpe"; then
         AC_RUN_IFELSE(
           [AC_LANG_SOURCE([[
-#include <stdlib.h>
+#include <stdlib.h> /* for exit() */
 #include <signal.h>
+#if !(defined _WIN32 && !defined __CYGWIN__)
+#include <unistd.h> /* for _exit() */
+#endif
 
 static void
 sigfpe_handler (int sig)
